recipe/main_dish/curry.recipe

```diff
@@ -0,0 +1,8 @@
+Curry recipe:
```

Comment:
Please format correctly the recipe

Reply:
what do you mean ?
:((((

Reply:
There is no spaces after the number, each phrase should start with capital letter
```
3. Add them too
```